docs/reference/features.md

```diff
@@ -0,0 +1,61 @@
+# IF features
+
+This page simply lists the features of Impact Framework that are not plugins or CLI tools, along with a brief description, usage instruction and link to more detailed docs.
+
+Typically these features are enabled using a piece of manifest config.
+
+## Aggregate
+
+Aggregate collects and summarizes data across time or across components in your tree.
+
+### How to configure
+
+Add the following config to your manifest (this example is for aggregating "cpu/utilization" values across both time and components):
+
+```yaml
+aggregation:
+  metrics:
+    "cpu/utilization":
```

Comment:
```suggestion
    "cpu/utilization":
       method: "avg"  
    
```

Reply:
```suggestion
    "cpu/utilization":
```
Comment was correct at the time of review, but the aggregation config has since been updated - no need for this change anymore.